File: app/src/main/java/com/example/james/rms/Core/ServePath/DeliveryOrderServePath.java
package com.example.james.rms.Core.ServePath;

import com.example.james.rms.NetWork.ServeProfile;

/**
 * Created by Jamie on 15/6/2017.
 */

public class DeliveryOrderServePath {

    //Find
    public static String serve_findAll(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getDe_order_findAll();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String serve_findByPartyIdAndStatus(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getDe_order_findByPartyIdAndStatus();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String serve_findByOrderIdAndStatus(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getDe_order_findByOrderIdAndStatus();
        String serve_path = serve+path;
        return serve_path;
    }

    //Save
    public static String serve_save(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getDe_order_save();
        String serve_path = serve+path;
        return serve_path;
    }

    public static String serve_saveOrderAndItem(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getDe_saveOrderAndItem();
        String serve_path = serve+path;
        return serve_path;
    }

    //Update


    //Delete
    public static String serve_delete(){
        String serve = ServeProfile.getServe();
        String path = ServeProfile.getDe_order_delete();
        String serve_path = serve+path;
        return serve_path;
    }
}

File: app/src/main/java/com/example/james/rms/core/serve_path/DeliveryOrderServePath.kt
package com.example.james.rms.core.serve_path

import com.example.james.rms.network.ServeProfile

/**
 * Created by Jamie on 15/6/2017.
 */

object DeliveryOrderServePath {
    val serve:String
        get() = ServeProfile.serve

    //Find
    fun serve_findAll(): String {
        return serve + ServeProfile.de_order_findAll
    }

    fun serve_findByPartyIdAndStatus(): String {
        return serve + ServeProfile.de_order_findByPartyIdAndStatus
    }

    fun serve_findByOrderIdAndStatus(): String {
        return serve + ServeProfile.de_order_findByOrderIdAndStatus
    }

    //Save
    fun serve_save(): String {
        return serve + ServeProfile.de_order_save
    }

    fun serve_saveOrderAndItem(): String {
        return serve + ServeProfile.de_saveOrderAndItem
    }

    //Update


    //Delete
    fun serve_delete(): String {
        return serve + ServeProfile.de_order_delete
    }
}
